x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_details.tsx

```diff
@@ -63,14 +71,14 @@ const ExceptionDetailsComponent = ({
     <StyledExceptionDetails grow={2}>
       <EuiFlexGroup direction="column" alignItems="flexStart">
         <EuiFlexItem grow={1}>
-          <EuiDescriptionList
-            compressed
-            type="column"
-            listItems={descriptionList}
-            titleProps={{ className: 'listTitle--width' }}
-            descriptionProps={{ className: 'listDescription--width' }}
-            data-test-subj="exceptionsViewerItemDetails"
-          />
+          <EuiDescriptionList compressed type="column" data-test-subj="exceptionsViewerItemDetails">
+            {descriptionListItems.map((item) => (
+              <Fragment key={item.title.toString()}>
```

Comment:
nit: you could also use a template literal here:
```suggestion
              <Fragment key={`${item.title}`}>>
```

Reply:
Thanks, updated! That looks nicer and for whatever reason feels better :)